sw/airborne/firmwares/rotorcraft/guidance/guidance_h.c

```diff
@@ -489,8 +490,24 @@ static void guidance_h_update_reference(void)
   }
 }
 
-#define MAX_POS_ERR   POS_BFP_OF_REAL(16.)
-#define MAX_SPEED_ERR SPEED_BFP_OF_REAL(16.)
+// GUIDANCE_H_MAX_POS_ERR_METER sets how much distance the drone tries to overcome in the horizontal loop at the same time
```

Comment:
Please use doxygen comments so that the description of the define shows up in the generated documentation....